### PR TITLE
Chore: Fix typos in docs

### PR DESCRIPTION
**Pull request by rohanKanojia**
I was reading your docs, I found some mistakes. Hope you find them useful!

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 You can grab the latest nightly binary executable for:
  * [Max OS X](https://storage.cloud.google.com/knative-nightly/client/latest/kn-darwin-amd64)
- * [Linix AMD 64](https://storage.googleapis.com/knative-nightly/client/latest/kn-linux-amd64)
+ * [Linux AMD 64](https://storage.googleapis.com/knative-nightly/client/latest/kn-linux-amd64)
```
Comment (rhuss) [inline docs/README.md]: Same as #181 .

**Comment by duglin** [comment]
@rohanKanojia CLA needs to be signed

**Comment by rohanKanojia** [comment]
@duglin : I remember I signed it as an individual. I tried signing it again and now it says it's already signed. Could you please check on your side ?

**Comment by duglin** [comment]
well, now it's happy :-)
/lgtm

**Pull request merged**